charts/fluid-dataloader/alluxio/templates/job.yaml

```diff
@@ -49,6 +49,7 @@ spec:
         role: dataload-pod
         app: alluxio
         targetDataset: {{ required "targetDataset should be set" .Values.dataloader.targetDataset }}
+        fluid.io/managed-by: fluid
```

Comment:
Can we reuse the helm helper function defined in the library helm chart?

Reply:
do you mean "library.fluid.labels" ? If so, will fix later.